nucypher/cli/commands/deploy.py

```diff
@@ -605,3 +605,28 @@ def transfer_ownership(general_config, actor_options, target_address, gas):
     else:
         receipts = ADMINISTRATOR.relinquish_ownership(new_owner=target_address, transaction_gas_limit=gas)
         emitter.ipc(receipts, request_id=0, duration=0)  # TODO: #1216
+
+
+@deploy.command("set-range")
+@group_general_config
+@group_actor_options
+@click.option('--minimum', help="Minimum value for range (in wei)", type=WEI)
+@click.option('--default', help="Default value for range (in wei)", type=WEI)
+@click.option('--maximum', help="Maximum value for range (in wei)", type=WEI)
+def set_range(general_config, actor_options, minimum, default, maximum):
+    """
+    Set the allowed range for the minimum reward rate in the policy manager contract.
+    """
+    emitter = general_config.emitter
+    ADMINISTRATOR, _, _, _ = actor_options.create_actor(emitter)
+
+    if not minimum:
+        minimum = click.prompt("Enter new minimum value for range", type=click.IntRange())
```

Comment:
Should we ensure non-negative number?

```suggestion
        minimum = click.prompt("Enter new minimum value for range", type=click.IntRange(min=0))
```

Reply:
✔️

Reply:
✔️